recipes/r-ggspatial/meta.yaml

```
@@ -0,0 +1,89 @@
+{% set version = '1.1.7' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ggspatial
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ggspatial_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ggspatial/ggspatial_{{ version }}.tar.gz
+  sha256: 83dd1e0936c71e772d69360a38a73f2c90f81d464596427b24c6efcebd786797
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-abind
+    - r-ggplot2 >=3.0.0
+    - r-glue
+    - r-rlang
+    - r-rosm >=0.2
+    - r-scales
+    - r-sf
+    - r-tibble
+    - r-tidyr
+  run:
+    - r-base
+    - r-abind
+    - r-ggplot2 >=3.0.0
+    - r-glue
+    - r-rlang
+    - r-rosm >=0.2
+    - r-scales
+    - r-sf
+    - r-tibble
+    - r-tidyr
+
+test:
+  commands:
+    - $R -e "library('ggspatial')"           # [not win]
+    - "\"%R%\" -e \"library('ggspatial')\""  # [win]
+
+about:
+  home: https://paleolimbot.github.io/ggspatial/, https://github.com/paleolimbot/ggspatial
+  license: LGPL-3.0-only
+  summary: Spatial data plus the power of the ggplot2 framework means easier mapping when input
+    data are already in the form of spatial objects.
+  license_family: LGPL
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/LGPL-3'
```

Where does LGPL come from? the cran meta data below says `License: GPL-3`
```suggestion
  license: GPL-3.0-only
  summary: Spatial data plus the power of the ggplot2 framework means easier mapping when input
    data are already in the form of spatial objects.
  license_family: GPL3
  license_file:
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
```